source/_integrations/lock.mqtt.markdown

```diff
@@ -250,7 +258,9 @@ mqtt:
   lock:
     - name: Frontdoor
       state_topic: "home-assistant/frontdoor/"
+      code_format: "^\\d{4}$"
       command_topic: "home-assistant/frontdoor/set"
+      command_template: '{ "{{ value }}": "{{ code }}" }'
```

Comment:
Is this template realistic, or should it better be something like:
`command_template: '{ "action": "{{ value }}", "code":"{{ code }}" }'`